### PR TITLE
update drone paths

### PR DESCRIPTION
**Pull request by cupcakearmy**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ steps:
       host: love-foundation.org
       user: root
       port: 1312
-      target: /srv/staging
+      target: /srv/lf/website//staging
```
Comment (moekify) [inline .drone.yml:53]: One `/` too much

**Review by moekify — CHANGES_REQUESTED** [review]
Tiny change :)

**Comment by cupcakearmy** [comment]
> Tiny change :)
> 
> And why is the commit label `use node 14`?

Becuase I'm a bonobo 😆

**Comment by moekify** [comment]
How about squashing & amending the commit message to make sense :) 

And no upsi in the history :D

**Comment by cupcakearmy** [comment]
> How about squashing & amending the commit message to make sense :)
> 
> And no upsi in the history :D

Github can squash on merge right?

**Comment by moekify** [comment]
Disabled atm, but that also doesn't change your commit message :)

Nicco ***@***.***> schrieb am Mi., 21. Juli 2021, 13:03:

**Pull request merged**